include/ceed.h

```diff
@@ -61,6 +61,10 @@
   static int name
 #endif
 
+#ifndef CEED_Q_VLA
+#  define CEED_Q_VLA Q
+#endif
```

Comment:
LGTM, but please document this with a comment here and a note in the documentation (VLA is a C99 feature that we find useful, but is not compatible with the C++ dialect used by CUDA.)

Reply:
I went ahead and added all of our macros to the documentation.